components/edit-collective/sections/HostVirtualCards.js

```diff
@@ -221,15 +220,8 @@ const HostVirtualCards = props => {
       </SettingsTitle>
       <Box>
         <SettingsSectionTitle>
-          <FormattedMessage id="Host.VirtualCards.Settings.Title" defaultMessage="Settings and Policy" />
+          <FormattedMessage id="Host.VirtualCards.Settings.Title" defaultMessage="Settings" />
```

Comment:
Why not but I don't see why it's better to have this section named "Settings" when it contains "Settings and Policy".

Reply:
@znarf : Yes, I've reverted this back as well. 😄